ras_backstage/resources/secure_messaging/get_thread_list.py

```diff
@@ -27,7 +27,7 @@ class GetThreadsList(Resource):
     def get(encoded_jwt):
         message_args = {
             'limit': request.args.get('limit', 1000),
-            'survey': request.args.get('survey')
+            'survey': request.args.getlist('survey')
```

Comment:
Will this have any other consequences? Looks nice and simple but are there no other side effects?

Reply:
The fix in secure-message should handle it